### PR TITLE
Various improvements

### PR DESCRIPTION
**Pull request by honzajavorek**
See commits for details. The subdomains stuff is untested :smirk:

### CODE REVIEW

**Comment by encukou** [comment]
IIRC, for subdomains you need to [set SERVER_NAME](https://github.com/pyvec/pyvo.cz/blob/master/pyvocz/app.py#L25)

**Comment by honzajavorek** [comment]
Added.

File: pythoncz/__init__.py [hunk 1]
```diff
@@ -25,4 +27,10 @@
 app.config.from_object(__name__)
 
 
+if app.debug:
+    app.config['SERVER_NAME'] = 'python.cz'
```
Comment (encukou) [inline pythoncz/__init__.py]: Shouldn't it be the other way around? `if app.debug` then use `0.0.0.0`?

Comment (honzajavorek) [inline pythoncz/__init__.py]: :facepalm:

**Pull request merged**